js/libs/keycloak-js/src/keycloak.js

```diff
@@ -133,8 +133,8 @@ function Keycloak (config) {
             }
 
             if (initOptions.pkceMethod) {
-                if (initOptions.pkceMethod !== "S256") {
-                    throw new TypeError(`Invalid value for 'pkceMethod', expected 'S256' but got '${initOptions.pkceMethod}'.`);
+                if (initOptions.pkceMethod !== "S256" || initOptions.pkceMethod !== false) {
```

Comment:
It should be `&&` not `||`.  
Furthermore this does not fix the bug, because when `initOptions.pkceMethod` is "false" the if clause in line 135 will fail and the `pkceMethod` will still be set to "S256" within the else section. So the if clause in line 135 should be: `if (initOptions.pkceMethod || initOptions.pkceMethod === false)`

Reply:
Yeah, was a bit to fast here. Pushed a fix.